persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_18/generated/medicalneed_client.bal

```diff
@@ -1,139 +0,0 @@
-// AUTO-GENERATED FILE. DO NOT MODIFY.
-
-// This file is an auto-generated file by Ballerina persistence layer for MedicalNeed.
-// It should not be modified by hand.
```

Comment:
Why do we have to remove this client?

Reply:
`generate_18` case is now a negative scenario. as we only allow bidirectional mapping in associations